Dockerfile.base

```diff
@@ -78,7 +78,10 @@ RUN apt-get -y update && \
 		valgrind \
 		wget \
 		ovmf \
-		xz-utils
+		xz-utils \
+		libboost-all-dev \
+		thrift-compiler \
+		libthrift-dev
```

Comment:
Technically speaking, only `thrift-compiler` is required for Zephyr builds. The other two are only required for testing against native applications.

Reply:
@stephanosio - what is your preference for the container? To add only thrift-compiler?

Reply:
I removed the additional packages, so only `thrift-compiler` is pulled in.

Reply:
Now that I think about it, I think `libthrift-dev` might still be necessary.